cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -150,8 +155,21 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
             sourceControlUsernameCredentials.name,
             mavenCredentialsUsername.name
         );
+        binaryApplication.perform_assessment("low", [stakeHolder]);
         binaryApplication.analyze();
         binaryApplication.verifyAnalysisStatus("Completed");
         binaryApplication.selectApplication();
     });
+
+    it("Assess application", function () {
+        const assessment = new Analysis(
```

Comment:
This should be application.

const application = new Analysis(

Reply:
@sshveta same is here, just a variable name